test/es6module/rlexe.xml

```diff
@@ -69,6 +69,12 @@
       <tags>exclude_sanitize_address</tags>
     </default>
   </test>
+  <test>
+    <default>
+      <files>dynamic_import_promises_5796.js</files>
+      <compile-flags>-ESDynamicImport</compile-flags>
```

Comment:
Would you mind adding `<tags>exclude_jshost</tags>` to this test and also the test for 3257? These are failing with our internal test host. I can update our host later but I don't see a reason to block your PR for it.

Reply:
done

Reply:
Thank you, I'll merge this in once the CI passes.